src/System.Private.CoreLib/shared/System/Collections/Generic/IAsyncEnumerator.cs

```diff
@@ -2,6 +2,7 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+#nullable enable
```

Comment:
TODO-NULLABLE-GENERIC comment in this file? T should be marked as MaybeNull/Nullable when available.

Reply:
Can you clarify, Santi?  What would you expect to be annotated here?

Reply:
Shouldn't T me marked as MaybeNull here? I.e to state that `T Current` could return null?

Reply:
It could only return null if:
- the type was instantiated with a nullable T, in which case it's already annotated appropriately
- Current was used in an unsupported way, e.g. before calling MoveNext, in which case behavior is explicitly undefined

Reply:
Ok, cool, thanks 😄